woocommerce/woocommerce-gutenberg-products-block
# Move mini cart outside cart-checkout folder

This is a second attempt at this, as the [previous PR](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6126) caused a [nasty merge conflict](p1649150426224029-slack-C02UBB1EPEF). This PR moves the `mini-cart` and  `shared` folders outside the `cart-checkout` folder and removes this. The `cart` and `checkout` blocks have already been moved outside of this folder, so it doesn't make sense anymore.

This also gets rid of some special webpack config needed to specify a custom directory instead of matching the regex used for all other blocks.


### Manual Testing

How to test the changes in this Pull Request:

1. Smoke test the Cart and make sure everything shows up, including all inner blocks
2. Smoke tests the Mini Cart block in the same way
